analysis/model/src/main/kotlin/de/maibornwolff/codecharta/serialization/ProjectDeserializer.kt

```diff
@@ -32,15 +32,14 @@ package de.maibornwolff.codecharta.serialization
 import com.google.gson.GsonBuilder
 import de.maibornwolff.codecharta.model.Node
 import de.maibornwolff.codecharta.model.Project
-import java.io.BufferedReader
-import java.io.FileNotFoundException
-import java.io.FileReader
-import java.io.Reader
+import mu.KotlinLogging
+import java.io.*
 
 /**
  * This class provides static methods and functions to convert a json to a Project-Object
  */
 object ProjectDeserializer {
```

Comment:
Done. (Since the Deserializer is in fact an Object and not a class i did not do it with the overloading of the construtor, however I overloaded the deserializeProject() method, and i also simplified the class, as we had unused functions in it)